src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -228,6 +244,16 @@ private void ResetBinding(bool abortCurrentlyRunningWorklows, bool clearCurrentB
             }
         }
 
+        private void UpdateSharedBindingSuggestion()
+        {
+            SharedBindingConfig = sharedBindingConfigProvider.GetSharedBinding();
+
+            VisualStateManager.HasSharedBinding = SharedBindingConfig != null;
+            VisualStateManager.ResetConnectionConfiguration();
+
+            sharedBindingSuggestionService.Suggest(SharedBindingConfig.GetServerType(),() => this.ActiveSection?.ConnectCommand);
```

Comment:
ConnectCommand will be replaced by non-null value after initialization. In order for delayed execution to pick that up, I used a lambda